### PR TITLE
mainPage and fixes

### PR DESCRIPTION
**Pull request by koprjukki**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -43,13 +43,20 @@ const App = () => {
...
-          </GuestRoute>
+
+        {/* Only guests routes */}
+        <GuestRoute path={ROUTES.REGISTRATION_ROUTE}>
+          <AsyncRegistration />
+        </GuestRoute>
+        <GuestRoute path={ROUTES.LOGIN_ROUTE}>
+          <AsyncLogin />
+        </GuestRoute>
+        <GuestRoute path={ROUTES.CREATE_TRANSACTION}>
```
Comment (an-parubets) [inline src/App.js]: PrivateRoute

File: src/components/Balance/Balance.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ export const Balance = ({ balance }) => {
   return (
     <div className={style.balance}>
       <p>Баланс</p>
-      <span>{balance}</span>
+      <span>{balance}.00 грн</span>
```
Comment (an-parubets) [inline src/components/Balance/Balance.js]: {balance.toFixed(2)}

File: src/App.js (hunk 1 above)
Comment (koprjukki) [reply src/App.js]: fixed

File: src/components/Balance/Balance.js (hunk 2 above)
Comment (koprjukki) [reply src/components/Balance/Balance.js]: fixed

**Pull request merged**